packages/server/app/services/reaction.service.ts

```diff
@@ -0,0 +1,106 @@
+import { Op } from 'sequelize'
+import {
+    Reaction,
+    ReactionAttributes,
+    type ReactionCreationAttributes,
+} from '../models/reaction.model'
+import type { BaseService } from './base.service'
+import { groupBy } from '../utils/ArrayFunc'
+import { Emoji } from '../models/emoji.model'
+
+export type CreateRequest = ReactionCreationAttributes
```

Comment:
Так как тип экспортируемый, то лучше использовать более специфичное имя

```suggestion
export type ReactionCreateRequest = ReactionCreationAttributes
```